webapp/src/main/webapp/js/brandingColors.js

```diff
@@ -0,0 +1,394 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+
+
+let theme = "themes/wilma/";
+
+const colorNameConvertor = {
+    "primary-lighter": "themePrimaryColorLighter",
+    "primary-base": "themePrimaryColor",
+    "primary-darker": "themePrimaryColorDarker",
+    "banner-base": "themeBannerColor",
+    "secondary-base": "themeSecondaryColor",
+    "accent-base": "themeAccentColor",
+    "text-base": "themeTextColor",
+    "link-base": "themeLinkColor"
+}
+
+const colorNameConvertorInverse = Object.keys(colorNameConvertor).reduce((acc, key) => {
+        acc[colorNameConvertor[key]] = key;
+        return acc;
+    }, {})
+
+
+const defaultColorPalete = {
+    "themes/wilma/": {
+        "primary-lighter": "#7bb3cb",
+        "primary-base": "#2485ae",
+        "primary-darker": "#064d68",
+        "banner-base": "#023048",
+        "secondary-base": "#398aac",
+        "accent-base": "#749a02",
+        "text-base": "#5f6464",
+        "link-base": "#2485ae"
+    },
+    "themes/vitro/": {
+        "primary-lighter": "#62b6d7",
+        "primary-base": "#47b6d0",
+        "primary-darker": "#006279",
+        "banner-base": "#012d3d",
+        "secondary-base": "#398aac",
+        "accent-base": "#0ea5c8",
+        "text-base": "#595B5B",
+        "link-base": "#47b6d0"
+    },
+    "themes/tenderfoot/": {
```

Comment:
Hardcoding theme paths is not an option. It will be broken once theme is renamed for customization.